style.css

```diff
@@ -230,13 +231,96 @@ button {
 }
 
 .but {
-  /* stylelint-disable-next-line color-hex-length */
-  background-color: #ffffff;
+  background-color: #fff;
   margin: 5px 3px;
   color: #3dc58c;
   border-radius: 5px;
 }
 
+/* add form */
+
+#contact-form {
+  display: flex;
+  flex-direction: column;
+  background-image: url(Frame.png);
+  background-position-y: 96%;
+  background-position-x: 100%;
+  background-repeat: no-repeat;
+}
+
+#form {
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  margin-top: 2rem;
+  width: 90%;
+}
+
+/* #form-element li {
+  width: 90%;
+  padding: 0.5rem 0;
+} */
+
+input {
+  box-sizing: border-box;
+  height: 3em;
+  width: 100%;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  margin: 1rem auto;
+}
+
+input::placeholder {
+  font-style: italic;
+}
+
+textarea {
+  box-sizing: border-box;
+  border: 1px solid #60c095;
+  border-radius: 4px;
+  height: 10em;
+  width: 100%;
+  margin: 1rem auto;
+}
+
+input:hover,
+textarea:hover,
 button:hover {
-  background-color: #374e44;
+  border: 2px solid rgb(40, 184, 246);
+}
+
+#form button {
+  display: block;
+  color: #fff;
+  margin: auto;
+  border-radius: 4px;
+  height: 3.5rem;
+  font-size: 20px;
+}
+
+#form button:enabled {
+  background-color: #60c095;
+}
+
+#form button:active {
+  background-color: #3e8e41;
 }
```

Comment:
As requested by the previous code reviewer please consider using the same style for hover and press functionality as used in the Figma design rather than using your own set of styles as it's a project requirement to adopt the exact same styles. 

## Solution and Further Guidance:-

You can use these styles which I copied from Figma by clicking over the element and copying the styles from the right inspect sidebar.

```
#form button:hover {
   background: #36B37F;
   box-shadow: 0px 8px 16px rgba(54, 179, 127, 0.24);
   border-radius: 4px;

}

#form button:active {
    background-color: #008552;
}
```

![image](https://user-images.githubusercontent.com/94223367/228338746-4ef88599-dd11-493c-b77d-93fb2f861faf.png)